build/ci/job-template.yml

```diff
@@ -67,8 +67,14 @@ jobs:
     steps:
     # Extra MacOS step required to install OS-specific dependencies
     - ${{ if and(contains(parameters.pool.vmImage, 'macOS'), not(contains(parameters.name, 'cross')))  }}:
-      - script: export HOMEBREW_NO_INSTALLED_DEPENDENTS_CHECK=TRUE && brew install $(Build.SourcesDirectory)/build/libomp.rb --build-from-source --formula
+      - script: |
```

Comment:
I think we need a similar fix to https://github.com/dotnet/machinelearning/blob/7c32fa61485bcb0ce610921d37ef86771cbc8edc/build/vsts-ci.yml#L145 for the official build.